Documentation/spelling_wordlist.txt

```diff
@@ -453,6 +453,7 @@ kubespray
 kvstore
 kvstoremesh
 kvstores
+labelled
```

Comment:
I believe `labeled` is the blessed (American) spelling, ideally we should be consistent with this.

Reply:
Ah! Thanks, this explains why I had to add it to the wordlist. Will fix after a rebase